diff --git a/.prettierignore b/.prettierignore
new file mode 100644
index 0000000000000000000000000000000000000000..72e8ffc0db8aad71a934dd11e5968bd5109e54b4
--- /dev/null
+++ b/.prettierignore
@@ -0,0 +1 @@
+*
diff --git a/build/src/reconnecting-eventsource.d.ts b/build/src/reconnecting-eventsource.d.ts
index e524a4ec03472ae43d518cba62d0298084c89e37..fcbd42a1382c6525bacf475f07e0c6deffb50b85 100644
--- a/build/src/reconnecting-eventsource.d.ts
+++ b/build/src/reconnecting-eventsource.d.ts
@@ -1,7 +1,7 @@
 /// <reference types="node" />
 export interface ReconnectingEventSourceInit extends EventSourceInit {
     max_retry_time?: number;
-    eventSourceClass?: typeof EventSource;
+    eventSourceClass?: Class<EventSource>;
     lastEventId?: string;
 }
 export declare class EventSourceNotAvailableError extends Error {
